banzai/data.py

```diff
@@ -302,6 +311,13 @@ def data_section(self, section):
         self.meta['DATASEC'] = section.to_region_keyword()
         self._data_section = section
 
+    @property
+    def n_sub_exposures(self):
+        n_exposures = self.meta.get('NSUBREAD', 1)
+        if str(n_exposures).lower() in ['n/a', 'unknown', 'none', '']:
```

Comment:
I'm not sure what the full list of possible `NSUBREAD` values can be, but should this have a test for values that cannot be converted to Int?

Reply:
Good idea to put a float in like 1.5 - now what? Shoudl be a reject for invalid value

Reply:
Right now, if there is something pathological in the data like nsubread=1.5, we will crash out and raise an error which I think is the right behavior. The cases here are the ones we assume we are ok and can safely treat things as 1 subread.